src/Vat.php

```diff
@@ -24,10 +29,9 @@ public function countryInEurope(string $countryCode): bool
         return $this->countries->isCountryValid($countryCode) && $this->countries->isCountryInEu($countryCode);
```

Comment:
Can a vat number not be valid if it is not in EU? (maybe method name is a bit confusing if the logic is correct.)

Reply:
No it cannot be checked at the EU based API. But as this function says, it check wether a country is within Europe. The implementer needs this information to pick the correct rate.